en/hubs_tos.md

```diff
@@ -1,22 +1,17 @@
 # Terms of Service for Hubs
 
-Version 1.0, Effective October 11, 2022
-{: datetime="2022-10-11" }
+Version 1.0, Effective October 25, 2022
```

Comment:
Should we update the version number?

Reply:
no need. We never launched version 1. Just delayed it.